view/frontend/web/instantsearch.js

```diff
@@ -477,12 +477,10 @@ requirejs(['algoliaBundle', 'Magento_Catalog/js/price-utils'], function (algolia
 				return ['rangeInput', {
 					container: facet.wrapper.appendChild(createISWidgetContainer(facet.attribute)),
 					attribute: facet.attribute,
-					labels: {
-						currency: algoliaConfig.currencySymbol,
-						separator: algoliaConfig.translations.to,
-						button: algoliaConfig.translations.go
-					},
-					templates: templates,
+					templates: Object.assign({
```

Comment:
`Object.assign()` is not supported in IE11. As we support Magento version 2.3 which does support IE11, I would recommend that you use the jQuery's `$.extend()` to merge the objects together.